src/public/sass/application.scss

```diff
@@ -1,6 +1,89 @@
 $govuk-global-styles: true;
 
-@import 'govuk-frontend/govuk/all';
+@import "govuk-frontend/govuk/all";
 
 // Patterns that aren't in govuk-frontend
-@import "patterns/related-items";
\ No newline at end of file
+@import "patterns/related-items";
+
+.interruption-card {
+  padding: 2em 3em;
+  background-color: govuk-colour("blue");
+
+  &.interruption-card-paper {
+    background-color: govuk-colour("blue");
+  }
+  @include govuk-media-query($until: tablet) {
+    background-image: none !important;
+  }
+
+  .govuk-button {
+    background-color: govuk-colour("white");
+    color: govuk-colour("black");
+
+    &:hover {
+      background-color: govuk-colour("mid-grey");
+    }
+  }
+
+  h1 {
+    color: govuk-colour("white");
+  }
+
+  .govuk-body,
+  .govuk-body-l {
+    color: govuk-colour("white");
+  }
+
+  ul {
+    color: govuk-colour("white");
+    width: 75%;
+    @include govuk-media-query(mobile) {
+      width: 100%;
+      margin-top: 0.5em;
+    }
+
+    li {
+      h2 {
+        padding-bottom: 0.5em;
+        display: inline-block;
+        width: 85%;
+        margin: 0 0 0 0.2em;
+        @include govuk-media-query(mobile) {
+          width: 80%;
+        }
+      }
+
+      span {
+        vertical-align: top;
+      }
+    }
+  }
+
+  .govuk-link.govuk-link--interrupt {
+    color: govuk-colour("black");
+  }
+
+  .govuk-link {
+    color: govuk-colour("white");
+  }
+
+  .button {
+    background-color: govuk-colour("white");
+    color: $govuk-link-colour;
+    font-weight: 700;
+
+    &:focus,
+    &:hover {
+      background-color: govuk-colour("light-grey");
+      color: $govuk-link-colour;
+    }
+  }
+
+  .circle-white {
+    background-color: govuk-colour("white");
+    color: govuk-colour("black");
+    margin-right: 0.5em;
+    position: relative;
+    top: -0.3em;
+  }
+}
```

Comment:
I was wondering if maybe we should store these in a separate scss file under the patterns directory and then in the applications.scss we can just import them? Curious what you think @franciscog-kainos

Reply:
Good point